sdk/spring/azure-spring-boot-starter-keyvault-certificates/src/main/java/com/azure/spring/security/keyvault/certificates/starter/KeyVaultCertificatesEnvironmentPostProcessor.java

```diff
@@ -3,112 +3,105 @@
 package com.azure.spring.security.keyvault.certificates.starter;
 
 import com.azure.security.keyvault.jca.KeyVaultJcaProvider;
+import com.azure.security.keyvault.jca.KeyVaultKeyStore;
 import com.azure.security.keyvault.jca.KeyVaultTrustManagerFactoryProvider;
-
-import java.security.Security;
-import java.util.Properties;
-import javax.net.ssl.HttpsURLConnection;
-
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.env.EnvironmentPostProcessor;
 import org.springframework.core.annotation.Order;
 import org.springframework.core.env.ConfigurableEnvironment;
 import org.springframework.core.env.MutablePropertySources;
 import org.springframework.core.env.PropertiesPropertySource;
+import org.springframework.util.StringUtils;
 
-import static org.springframework.core.Ordered.LOWEST_PRECEDENCE;
+import javax.net.ssl.HttpsURLConnection;
+import java.security.Security;
+import java.util.Optional;
+import java.util.Properties;
 
 /**
  * Leverage {@link EnvironmentPostProcessor} to add Key Store property source.
  */
-@Order(LOWEST_PRECEDENCE)
+@Order
```

Comment:
why are we removing the LOWEST_PRECEDENCE.

Reply:
`LOWEST_PRECEDENCE` is the default value. I just accept Intellij's suggestion.